issue_maker.py

```diff
@@ -293,14 +293,17 @@ def update():
             with open('unsupported.txt', 'a') as f:
                 f.write('{song} by {artist}\n'.format(song=song, artist=artist))
 
-            issue = create_issue(song, artist, version, stripped)
-            if issue['status_code'] == 201:
-                print('Created issue on the GitHub repo for {song} by {artist}.'.format(song=song, artist=artist))
-                return 'Lyrics for that song may not exist on Genius. Created issue on the GitHub repo for {song} by ' \
-                       '{artist} to investigate further. \n{link}'.format(
-                        song=song, artist=artist, link=issue['link'])
+            if re.fullmatch(r'[A-Z|a-z|\s]+', song) and re.fullmatch(r'[A-Z|a-z|\s]+', artist):
+                return 'Lyrics of {song} by {artist} may not exist on Genius.'.format(song=song, artist=artist)
```

Comment:
You could also include an "If you feel there's an error" text as seen on https://github.com/SwagLyrics/swaglyrics-backend/blob/60e868191e937920bd459b7299d3464dd59217a0/issue_maker.py#L308

Reply:
I see, I will work on it.

Reply:
Sure. Rest everything looks good to me 👍